Get-PoshThemes error - Cannot bind argument to parameter 'Path' because it is null

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

When calling Get-PoshThemes I get an error before the initial output: 

```
> Get-PoshThemes
Resolve-Path:
Line |
 308 |          $Path = (Resolve-Path -Path $temp).ProviderPath
     |                                      ~~~~~
     | Cannot bind argument to parameter 'Path' because it is null.
   __  _____ _      ___  ___       ______         _      __
  / / |  _  | |     |  \/  |       | ___ \       | |     \ \
 / /  | | | | |__   | .  . |_   _  | |_/ /__  ___| |__    \ \
< <   | | | | '_ \  | |\/| | | | | |  __/ _ \/ __| '_ \    > >
 \ \  \ \_/ / | | | | |  | | |_| | | | | (_) \__ \ | | |  / /
  \_\  \___/|_| |_| \_|  |_/\__, | \_|  \___/|___/_| |_| /_/
                             __/ |
                            |___/

```


The command continues without errors but something must be wrong since the error shows up.
The error is the same if I call _Get-PoshThemes -List_ and it is also same error if I supply a value in path: 

```
PS> Test-Path $poshThemePath
True
PS> dir $poshThemePath

        Directory: C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
la---        20-09-2022     12:01            129 󰉢  faketheme.yaml
la---        20-09-2022     12:01              0   faketheme.yaml.bak
la---        16-10-2023     21:43           5010   montys_jgj.omp.json
la---        20-09-2022     11:17              0 󰈙  test.txt

PS> Test-Path $poshThemePath
True
PS> $poshThemePath
C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme
PS> Get-PoshThemes -Path $poshThemePath
Resolve-Path:
Line |
 308 |          $Path = (Resolve-Path -Path $temp).ProviderPath
     |                                      ~~~~~
     | Cannot bind argument to parameter 'Path' because it is null.
   __  _____ _      ___  ___       ______         _      __
  / / |  _  | |     |  \/  |       | ___ \       | |     \ \
 / /  | | | | |__   | .  . |_   _  | |_/ /__  ___| |__    \ \
< <   | | | | '_ \  | |\/| | | | | |  __/ _ \/ __| '_ \    > >
 \ \  \ \_/ / | | | | |  | | |_| | | | | (_) \__ \ | | |  / /
  \_\  \___/|_| |_| \_|  |_/\__, | \_|  \___/|___/_| |_| /_/
                             __/ |
                            |___/
Theme: montys_jgj

   󰋜  0ms  15:28  󰲝   


```

Screendump of the error occuring:
![oh_my_posh_getposhthemes_error](https://github.com/JanDeDobbeleer/oh-my-posh/assets/42742575/4ae812f9-567d-4bb0-a314-5e25a3db4723)


### Theme

{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "background": "#003543",
          "foreground": "#00c983",
          "leading_diamond": "\ue0b6",
          "style": "diamond",
          "type": "os"
        },
        {
          "background": "#DA627D",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "folder_icon": "\uf115",
            "folder_separator_icon": "\\",
            "home_icon": "\udb80\udedc",
            "mapped_locations": {
              "C:\\Dev\\Git": "\uf1d3 ",
              "C:\\Dev\\Git\\GitLab": "\uf1d3 \uea9f \uf296 ",
              "C:\\Dev\\Git\\GitHub": "\uf1d3 \uea9f \uea84 ",
              "C:\\Users\\jgj.it\\OneDrive - CBS - Copenhagen Business School" : "\ue762",
              "C:\\Users\\jgj.it\\Dropbox" : "\ue707"

            },
            "style": "agnoster_full",
            "windows_registry_icon": "\ue0b1"
          },
          "style": "powerline",
          "type": "path"
        },
        {
          "background": "#FCA17D",
          "background_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}#FFEB3B{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}#FFCC80{{ end }}",
            "{{ if gt .Ahead 0 }}#B388FF{{ end }}",
            "{{ if gt .Behind 0 }}#B388FB{{ end }}"
          ],
          "foreground": "#ffffff",
          "foreground_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}#444{{ end }}"
          ],
          "powerline_symbol": "\ue0b0",
          "properties": {
            "branch_icon": " <#ffffff>\ue0a0 </>",
            "display_status": true,
            "fetch_stash_count": true,
            "fetch_status": true,
            "fetch_upstream_icon": true
          },
          "style": "powerline",
          "type": "git"
        },
        {
          "background": "#043452",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "style": "powerline",
          "type": "poshgit"
        },
        {
          "background": "#76b367",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "style": "powerline",
          "type": "node"
        },
        {
          "background": "#83769c",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "always_enabled": true,
            "style" : "round"
          },
          "style": "powerline",
          "type": "executiontime"
        },
        {
          "background": "#33658A",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "time_format": "15:04"
          },
          "style": "powerline",
          "type": "time"
        },
        {
          "type": "connection",
          "style": "powerline",
          "background": "#22eead",
          "foreground": "#222222",
          "powerline_symbol": "\uE0B0",
          
          "template": "{{ if eq .Type \"wifi\"}} \uf1eb  {{ .SSID }}{{ else if eq .Type \"ethernet\"}} \udb83\udc9d {{ end }}"
          
        },
        {
          "type": "wifi",
          "style": "powerline",
          "background": "#8822ee",
          "foreground": "#222222",
          "background_templates": [
            "{{ if (lt .Signal 60) }}#DDDD11{{ else if (lt .Signal 90) }}#DD6611{{ else }}#11CC11{{ end }}"
          ],
          "powerline_symbol": "\uE0B0",
          "template": "\uFAA8 {{ .SSID }} {{ .Signal }}% {{ .ReceiveRate }}Mbps\u2800"
        },
        {
          "type": "project",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#193549",
          "background": "#ffeb3b",
          "template": " {{ if .Error }}{{ .Error }}{{ else }}{{ if .Version }}\uf487 {{.Version}}{{ end }} {{ if .Name }}{{ .Name }}{{ end }}{{ if .Target }} ({{ .Target }}){{ end }}{{ end }} "
        },
        {
          "background": "#2e9599",
          "background_templates": [
            "{{ if gt .Code 0 }}red{{ end }}"
          ],
          "template": " {{ if gt .Code 0 }}\uf00d {{ .Code }}{{ else }}\uf42e{{ end }} ",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "always_enabled": true
          },
          "style": "diamond",
          "trailing_diamond": "\ue0b4",
          "type": "exit"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "foreground": "#cd5e42",
          "style": "plain",
          "type": "root"
        },
        {
          "foreground": "#CD4277",
          "style": "plain",
          "type": "text"
        }
      ],
      "type": "prompt"
    }
  ],
  "final_space": true,
  "version": 2
}


### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 18.15.0

Shell: pwsh (7.3.8)

Prompt:

┬»├®├é ┬»├┐┬¼ ┬»├®Ôûæ ┬¥Ôûæ├»┬ú ┬»├®Ôûæ 0ms ┬»├®Ôûæ 15:32 ┬»├®Ôûæ ┬¥ÔûæÔûô├ÿ ┬»├®Ôûæ ┬┤├ë┬½ ┬»├®Ôöñ 

Segments:

ConsoleTitle(false)                        -   0 ms
Os(true)                                   -   0 ms
Path(true)                                 -   3 ms
Git(false)                                 -  21 ms
Poshgit(false)                             -   0 ms
Node(false)                                -   0 ms
Executiontime(true)                        -   1 ms
Time(true)                                 -   1 ms
Connection(true)                           -   4 ms
Wifi(false)                                -   0 ms
Project(false)                             -   5 ms
Exit(true)                                 -   2 ms
Root(false)                                -   0 ms
Text(false)                                -   0 ms

Run duration: 35.6118ms

Cache path: C:\Users\jgj.it\AppData\Local\oh-my-posh

Config path: C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme\montys_jgj.omp.json

Logs:

[DEBUG] 15:32:13.029 shell.go:Getenv:392 ├ö├Ñ├å NO DATA
[TRACE] 15:32:13.029 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 15:32:13.029 shell.go:Getenv:392 ├ö├Ñ├å C:\Users\jgj.it\AppData\Local
[TRACE] 15:32:13.029 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 15:32:13.029 shell_windows.go:CachePath() - 504.1Ôö¼├üs
[DEBUG] 15:32:13.029 shell.go:Getenv:392 ├ö├Ñ├å C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme\montys_jgj.omp.json
[TRACE] 15:32:13.029 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 15:32:13.029 shell.go:Shell:646 ├ö├Ñ├å no shell name provided in flags, trying to detect it
[DEBUG] 15:32:13.039 shell.go:Shell:654 ├ö├Ñ├å process name: pwsh.exe
[TRACE] 15:32:13.039 shell.go:Shell() - 9.4488ms
[TRACE] 15:32:13.039 shell.go:resolveConfigPath() - 9.4488ms
[TRACE] 15:32:13.039 shell.go:Init() - 10.0759ms
[TRACE] 15:32:13.039 shell.go:Flags() - 0s
[TRACE] 15:32:13.040 config.go:loadConfig() - 1.1137ms
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[DEBUG] 15:32:13.040 shell.go:Getenv:392 ├ö├Ñ├å NO DATA
[TRACE] 15:32:13.040 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 15:32:13.040 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.040 shell_windows.go:WindowsRegistryKeyValue:215 ├ö├Ñ├å ColorizationColor(DWORD): 0xC44967AA
[TRACE] 15:32:13.040 shell.go:Shell() - 0s
[DEBUG] 15:32:13.040 shell.go:Getenv:392 ├ö├Ñ├å 7.3.8
[TRACE] 15:32:13.040 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 15:32:13.040 debug.go:PrintDebug:22 ├ö├Ñ├å Segment: Title
[DEBUG] 15:32:13.040 text.go:Render:70 ├ö├Ñ├å Rendering template: 
[DEBUG] 15:32:13.040 shell.go:Getenv:392 ├ö├Ñ├å 46
[TRACE] 15:32:13.040 shell.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 15:32:13.040 shell.go:Getenv:392 ├ö├Ñ├å 1
[TRACE] 15:32:13.040 shell.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[TRACE] 15:32:13.040 shell.go:Shell() - 0s
[TRACE] 15:32:13.040 shell.go:Shell() - 0s
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[DEBUG] 15:32:13.040 properties.go:GetString:28 ├ö├Ñ├å {{ .Code }}
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.040 shell.go:Pwd:422 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.040 shell.go:Pwd() - 0s
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[DEBUG] 15:32:13.040 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.040 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Exit
[TRACE] 15:32:13.040 shell.go:Pwd() - 0s
[TRACE] 15:32:13.040 shell.go:StatusCodes() - 0s
[DEBUG] 15:32:13.040 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[DEBUG] 15:32:13.040 text.go:Render:70 ├ö├Ñ├å Rendering template: {{ .Code }}
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.040 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Node
[DEBUG] 15:32:13.040 properties.go:GetStringArray:52 ├ö├Ñ├å extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[DEBUG] 15:32:13.040 properties.go:GetStringArray:52 ├ö├Ñ├å folders: []
[DEBUG] 15:32:13.040 properties.go:GetBool:22 ├ö├Ñ├å home_enabled: false
[TRACE] 15:32:13.040 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.040 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[TRACE] 15:32:13.040 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.040 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.040 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Git
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[TRACE] 15:32:13.040 shell.go:Pwd() - 0s
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[TRACE] 15:32:13.040 shell.go:Flags() - 0s
[TRACE] 15:32:13.040 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.040 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.040 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Time
[DEBUG] 15:32:13.040 properties.go:GetString:28 ├ö├Ñ├å 15:04
[DEBUG] 15:32:13.040 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.040 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.040 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Path
[TRACE] 15:32:13.040 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.041 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Executiontime
[TRACE] 15:32:13.041 shell.go:Flags() - 0s
[DEBUG] 15:32:13.041 properties.go:GetBool:22 ├ö├Ñ├å always_enabled: true
[TRACE] 15:32:13.041 shell.go:ExecutionTime() - 0s
[TRACE] 15:32:13.041 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.041 properties.go:GetFloat64:34 ├ö├Ñ├å threshold: 500.000000
[DEBUG] 15:32:13.041 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Os
[TRACE] 15:32:13.041 shell.go:Flags() - 0s
[DEBUG] 15:32:13.041 properties.go:GetString:28 ├ö├Ñ├å ┬»├┐┬¼
[TRACE] 15:32:13.041 shell.go:Pwd() - 0s
[TRACE] 15:32:13.041 shell.go:Flags() - 0s
[TRACE] 15:32:13.041 shell.go:Shell() - 0s
[TRACE] 15:32:13.041 shell.go:Flags() - 0s
[TRACE] 15:32:13.041 shell.go:Flags() - 0s
[TRACE] 15:32:13.041 shell.go:Flags() - 0s
[TRACE] 15:32:13.041 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.041 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[TRACE] 15:32:13.041 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.041 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Connection
[DEBUG] 15:32:13.041 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 properties.go:GetString:28 ├ö├Ñ├å wifi|ethernet
[DEBUG] 15:32:13.041 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Project
[DEBUG] 15:32:13.041 properties.go:GetBool:22 ├ö├Ñ├å mapped_locations_enabled: true
[TRACE] 15:32:13.041 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.041 properties.go:GetString:28 ├ö├Ñ├å ┬»├®ÔûÆ
[DEBUG] 15:32:13.041 properties.go:GetString:28 ├ö├Ñ├å ┬»├®ÔûÆ
[DEBUG] 15:32:13.041 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 properties.go:GetString:28 ├ö├Ñ├å ┬¥Ôûæ├»┬ú
[TRACE] 15:32:13.041 shell_windows.go:Root() - 0s
[TRACE] 15:32:13.041 shell.go:Shell() - 0s
[TRACE] 15:32:13.041 shell.go:StatusCodes() - 0s
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[TRACE] 15:32:13.041 shell_windows.go:IsWsl() - 0s
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 text.go:Render:70 ├ö├Ñ├å Rendering template: C:\Dev\Git\GitHub
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 text.go:Render:70 ├ö├Ñ├å Rendering template: C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 text.go:Render:70 ├ö├Ñ├å Rendering template: C:\Users\jgj.it\Dropbox
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.041 text.go:Render:70 ├ö├Ñ├å Rendering template: C:\Dev\Git
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[TRACE] 15:32:13.041 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.042 shell.go:TemplateCache:827 ├ö├Ñ├å environment: [ALLUSERSPROFILE=C:\ProgramData APPDATA=C:\Users\jgj.it\AppData\Roaming ChocolateyInstall=C:\ProgramData\chocolatey ChocolateyLastPathUpdate=133364866066501708 ChocolateyToolsLocation=C:\tools CommonProgramFiles=C:\Program Files\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPUTERNAME=CBS12192 ComSpec=C:\WINDOWS\system32\cmd.exe CONDA_PROMPT_MODIFIER=False DLMAS_HOME=C:\PROGRA~2\Red Gate\SQL Change Automation PowerShell\ DriverData=C:\Windows\System32\Drivers\DriverData GIT_SSH=TortoiseGitPlink GOPATH=C:\Users\jgj.it\go HOMEDRIVE=C: HOMEPATH=\Users\jgj.it JGJ_TEST=foo LOCALAPPDATA=C:\Users\jgj.it\AppData\Local LOGONSERVER=\\AD-DC-PRD-03 NUMBER_OF_PROCESSORS=16 OneDrive=C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School OneDriveCommercial=C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School OneDriveConsumer=C:\Users\jgj.it\OneDrive OPENSSL_CONF=C:\Program Files\OpenSSL-Win64\bin\openssl.cfg OS=Windows_NT Path=C:\Program Files\PowerShell\7;C:\Python312\Scripts\;C:\Python312\;C:\Python311\Scripts\;C:\Python311\;C:\Python310\Scripts\;C:\Python310\;C:\Program Files\Common Files\Microsoft Shared\Microsoft Online Services;C:\Program Files (x86)\Common Files\Microsoft Shared\Microsoft Online Services;C:\Users\jgj.it\AppData\Roaming\ActiveState\bin;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\System32\WindowsPowerShell\v1.0\;C:\windows\System32\OpenSSH\;C:\ProgramData\chocolatey\bin;C:\Program Files\Azure Data Studio\bin;C:\Program Files\dotnet\;C:\Program Files (x86)\oh-my-posh\bin;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn\;C:\Program Files\Microsoft VS Code\bin;C:\Program Files (x86)\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\tools\gsudo\Current;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\PowerShell\7-preview\preview;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\160\DTS\Binn\;C:\Program Files\Git\cmd;C:\Program Files\TortoiseGit\bin;C:\Program Files\Go\bin;C:\Program Files\PowerShell\7\;C:\Program Files\GitHub CLI\;C:\Program Files\nodejs\;C:\Users\jgj.it\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Azure Data Studio\bin;C:\Users\jgj.it\.dotnet\tools;C:\Users\jgj.it\AppData\Local\Programs\Fiddler;C:\Users\jgj.it\AppData\Local\gitkraken\bin;C:\Program Files\MKVToolNix;C:\tools\neovim\nvim-win64\bin;C:\Program Files\OpenSSL-Win64\bin;C:\Users\jgj.it\AppData\Local\Microsoft\WindowsApps;C:\Users\jgj.it\.dotnet\tools;C:\Users\jgj.it\AppData\Local\GitHubDesktop\bin;C:\Users\jgj.it\go\bin;C:\Users\jgj.it\AppData\Roaming\npm PATHEXT=.SCM;.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW;.CPL POSH_AZURE_ENABLED=False POSH_CURSOR_COLUMN=1 POSH_CURSOR_LINE=46 POSH_GIT_ENABLED=False POSH_INSTALLER=manual POSH_PID=12292 POSH_SHELL_VERSION=7.3.8 POSH_THEME=C:\Users\jgj.it\OneDrive - CBS - Copenhagen Business School\PoshTheme\montys_jgj.omp.json POSH_THEMES_PATH=C:\Program Files (x86)\oh-my-posh\themes POWERLINE_COMMAND=oh-my-posh POWERSHELL_DISTRIBUTION_CHANNEL=MSI:Windows 10 Enterprise PROCESSOR_ARCHITECTURE=AMD64 PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 141 Stepping 1, GenuineIntel PROCESSOR_LEVEL=6 PROCESSOR_REVISION=8d01 ProgramData=C:\ProgramData ProgramFiles=C:\Program Files ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PSModulePath=C:\Users\jgj.it\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\Windows\System32\WindowsPowerShell\v1.0\Modules;C:\Program Files (x86)\Microsoft SQL Server\150\Tools\PowerShell\Modules\;C:\PROGRA~2\Red Gate\SQL Change Automation PowerShell\Modules\ PUBLIC=C:\Users\Public SESSIONNAME=Console SystemDrive=C: SystemRoot=C:\WINDOWS TEMP=C:\Users\jgj.it\AppData\Local\Temp TMP=C:\Users\jgj.it\AppData\Local\Temp UATDATA=C:\WINDOWS\CCM\UATData\D9F8C395-CAB8-491d-B8AC-179A1FE1BE77 USERDNSDOMAIN=HHK.DK USERDOMAIN=HHK USERDOMAIN_ROAMINGPROFILE=HHK USERNAME=jgj.it USERPROFILE=C:\Users\jgj.it windir=C:\WINDOWS WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={574e775e-4f2a-5b96-ac1e-a2962a402336} WT_SESSION=df5ee05a-c1ac-42a6-8ca3-1306dea163d1 ZES_ENABLE_SYSMAN=1 ZZTOP=foobar ZZZ_test=JGJ3]
[TRACE] 15:32:13.042 shell.go:Pwd() - 0s
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[DEBUG] 15:32:13.042 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.042 shell.go:User:557 ├ö├Ñ├å jgj.it
[TRACE] 15:32:13.042 shell.go:User() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.042 shell.go:Host:569 ├ö├Ñ├å CBS12192
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:Host() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.042 shell.go:Getenv:392 ├ö├Ñ├å NO DATA
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:Getenv(SHLVL) - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:TemplateCache() - 1.8457ms
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:TemplateCache() - 618.8Ôö¼├üs
[TRACE] 15:32:13.042 shell.go:TemplateCache() - 618.8Ôö¼├üs
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:GOOS() - 0s
[TRACE] 15:32:13.042 shell.go:PathSeparator() - 0s
[TRACE] 15:32:13.042 shell.go:Pwd() - 0s
[TRACE] 15:32:13.042 shell_windows.go:IsWsl() - 0s
[TRACE] 15:32:13.042 shell.go:StackCount() - 0s
[TRACE] 15:32:13.042 shell.go:TemplateCache() - 0s
[DEBUG] 15:32:13.042 properties.go:GetBool:22 ├ö├Ñ├å always_enabled: true
[TRACE] 15:32:13.042 shell.go:TemplateCache() - 0s
[DEBUG] 15:32:13.042 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.042 shell.go:HasFiles(package.json) - 1.2001ms
[TRACE] 15:32:13.042 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.043 win32_windows.go:isWriteable:299 ├ö├Ñ├å not current user or in group
[DEBUG] 15:32:13.043 win32_windows.go:isWriteable:299 ├ö├Ñ├å not current user or in group
[DEBUG] 15:32:13.043 win32_windows.go:isWriteable:303 ├ö├Ñ├å current user is member of S-1-5-21-842925246-963894560-725345543-71233
[DEBUG] 15:32:13.043 win32_windows.go:isWriteable:311 ├ö├Ñ├┤
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 15:32:13.043 win32_windows.go:isWriteable:313 ├ö├Ñ├å user has write access
[TRACE] 15:32:13.043 shell_windows.go:DirIsWritable() - 1.1673ms
[TRACE] 15:32:13.043 shell.go:TemplateCache() - 0s
[DEBUG] 15:32:13.043 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.043 shell.go:HasFiles(Cargo.toml) - 586Ôö¼├üs
[TRACE] 15:32:13.043 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.043 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.043 shell.go:HasFiles(pyproject.toml) - 503.4Ôö¼├üs
[TRACE] 15:32:13.043 shell.go:Pwd() - 0s
[TRACE] 15:32:13.043 networks_windows.go:wifiNetwork() - 0s
[DEBUG] 15:32:13.044 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.044 shell.go:HasFiles(composer.json) - 621.7Ôö¼├üs
[TRACE] 15:32:13.044 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.044 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.044 shell.go:HasFiles(*.nuspec) - 504.3Ôö¼├üs
[TRACE] 15:32:13.044 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.045 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.045 shell.go:HasFiles(*.vbproj) - 657.7Ôö¼├üs
[TRACE] 15:32:13.045 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.045 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.045 shell.go:HasFiles(*.fsproj) - 0s
[TRACE] 15:32:13.045 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.046 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.046 shell.go:HasFiles(*.csproj) - 506.4Ôö¼├üs
[TRACE] 15:32:13.046 shell.go:Pwd() - 0s
[ERROR] 15:32:13.046 shell_windows.go:Connection:249 ├ö├Ñ├å Network type 'wifi' not found
[TRACE] 15:32:13.046 shell.go:TemplateCache() - 0s
[DEBUG] 15:32:13.046 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.046 shell.go:HasFiles(JuliaProject.toml) - 524.5Ôö¼├üs
[TRACE] 15:32:13.046 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.046 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.046 shell.go:HasFiles(Project.toml) - 0s
[TRACE] 15:32:13.046 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.047 shell.go:HasFiles:458 ├ö├Ñ├å false
[TRACE] 15:32:13.047 shell.go:HasFiles(*.psd1) - 520.9Ôö¼├üs
[DEBUG] 15:32:13.047 properties.go:GetBool:22 ├ö├Ñ├å always_enabled: false
[DEBUG] 15:32:13.050 shell.go:CommandPath:608 ├ö├Ñ├å C:\Program Files\Git\cmd\git.exe
[TRACE] 15:32:13.050 shell.go:CommandPath(git.exe) - 10.6727ms
[TRACE] 15:32:13.050 shell.go:HasCommand(git.exe) - 10.6727ms
[TRACE] 15:32:13.050 shell.go:Pwd() - 0s
[ERROR] 15:32:13.062 shell.go:HasParentFilePath:742 ├ö├Ñ├å CreateFile .git: The system cannot find the file specified.
[TRACE] 15:32:13.062 shell.go:HasParentFilePath(.git) - 11.2506ms
[DEBUG] 15:32:13.062 properties.go:GetBool:22 ├ö├Ñ├å fetch_bare_info: false
[DEBUG] 15:32:13.063 text.go:Render:70 ├ö├Ñ├å Rendering template:  {{ if gt .Code 0 }}┬┤├ç├¼ {{ .Code }}{{ else }}┬┤├ë┬½{{ end }} 
[DEBUG] 15:32:13.063 text.go:Render:70 ├ö├Ñ├å Rendering template:  {{ if .WSL }}WSL at {{ end }}{{.Icon}} 
[DEBUG] 15:32:13.063 text.go:Render:70 ├ö├Ñ├å Rendering template:  {{ .CurrentDate | date .Format }} 
[DEBUG] 15:32:13.063 text.go:Render:70 ├ö├Ñ├å Rendering template:  {{ .Path }} 
[DEBUG] 15:32:13.063 text.go:Render:70 ├ö├Ñ├å Rendering template: {{ if eq .Type "wifi"}} ┬┤├º┬¢  {{ .SSID }}{{ else if eq .Type "ethernet"}} ┬¥ÔûæÔûô├ÿ {{ end }}
[DEBUG] 15:32:13.063 text.go:Render:70 ├ö├Ñ├å Rendering template:  {{ .FormattedMs }} 
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: diamond
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: powerline
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: {{ if gt .Code 0 }}red{{ end }}
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template: diamond
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Flags() - 0s
[TRACE] 15:32:13.064 shell.go:Flags() - 0s
[TRACE] 15:32:13.064 shell.go:Pwd() - 0s
[TRACE] 15:32:13.064 shell.go:Pwd() - 0s
[DEBUG] 15:32:13.064 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[DEBUG] 15:32:13.064 shell_windows.go:56 ├ö├Ñ├å C:\Users\jgj.it
[TRACE] 15:32:13.064 shell.go:GOOS() - 0s
[TRACE] 15:32:13.064 shell.go:GOOS() - 0s
[DEBUG] 15:32:13.064 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Text
[DEBUG] 15:32:13.064 segment.go:SetEnabled:497 ├ö├Ñ├å Segment: Root
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell_windows.go:Root() - 517.2Ôö¼├üs
[DEBUG] 15:32:13.064 text.go:Render:70 ├ö├Ñ├å Rendering template:   
[TRACE] 15:32:13.064 shell.go:TemplateCache() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[TRACE] 15:32:13.064 shell.go:Shell() - 0s
[DEBUG] 15:32:13.064 shell.go:Getenv:392 ├ö├Ñ├å C:\Users\jgj.it\AppData\Local
[TRACE] 15:32:13.064 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 15:32:13.064 shell_windows.go:CachePath() - 0s
[TRACE] 15:32:13.064 shell.go:Flags() - 0s
```

--- a/src/shell/scripts/omp.ps1
+++ b/src/shell/scripts/omp.ps1
@@ -305,7 +305,7 @@ New-Module -Name "oh-my-posh-core" -ScriptBlock {
             $Path = Read-Host 'Please enter the themes path'
         }
 
-        $Path = (Resolve-Path -Path $temp).ProviderPath
+        $Path = (Resolve-Path -Path $Path).ProviderPath
 
         $logo = @'
    __  _____ _      ___  ___       ______         _      __

